MiniAn/minian_parameters.py

```diff
@@ -152,6 +152,12 @@ def __init__(self, danse_parameters):
         self.update_all_func_params()
         self.parameters[defs.Parameters.danse.ADVANCED_SETTINGS] = self.advanced_settings.copy()
 
+        self.params_cross_reg = {
+            'crossReg'     : self.parameters[defs.Parameters.danse.CROSS_REG],
```

Comment:
Do we need this parameter here? If the parameters are empty, we shouldn't do the cross registration, I guess. Or if you wanna keep, it has to be renamed to `"cross_reg"`

Reply:
Could you also move the setting above the advanced parameters?